ckanext/validation_schema_generator/views/__init__.py

```diff
@@ -0,0 +1,156 @@
+import json
```

Comment:
Why is this file in a submodule, instead of just being `ckanext/validation_schema_generator/views.py`?